src/p4p/server/raw.py

```diff
@@ -39,6 +39,12 @@ class Handler(object):
 
     Use of this as a base class is optional.
     """
+    def open(self, value, **kws):
+        """
+        Called each time an Open operation is performed on this Channel
+
+        :param value:  A Value, or appropriate object (see nt= and wrap= of the constructor).
+        """
```

Comment:
Should this (and the `post` and `close` methods) call `op.done(error='Not supported')`), as other methods do?
At the very least for consistency this one should have a `pass`, to be the same as the other methods that don't call that method.